docs/source/model_design/concept_models.rst

```diff
@@ -0,0 +1,18 @@
+================================================
+Vivarium Model Considerations and Best Practices
+================================================
+
+The purpose of this page is to document information that is relevant to designing Vivarium simulation models as a whole, rather than information specific to particular sub-components of Vivarium simulations. 
+
+Outline of topics:
+
+- Common random numbers 
+		- Between model scenarios
+		- Between model locations
+		- Between model parameters
+- Statistical distributions of uncertainty
+	- Uniform
+	- Normal/Truncated normal
+	- Beta
+	- Ensemble
```

Comment:
For the section on probability distributions, I was thinking it would be useful to organize it according to what we are using them for rather than by distribution type. For example:
* What types of distributions could I use to model uncertainty in
    - a relative risk?
    - a mean difference?
    - a proportion?
    - a cost estimate?

And so on.  And then also address questions like:
* What do I do if someone gives me a confidence interval that's very asymmetric (this can pose problems because you may not be able to find a distribution that matches your assumptions well)?
* How do we think about modeling uncertainty based on belief, with very little data available, vs. uncertainty based on combining multiple estimates from published papers?
* How do I model uncertainty if I am extrapolating a subnational estimate to use as a national estimate?

And whatever other methodological questions we run into.

For quick reference, it may also be useful to compile a list of distributions we find ourselves commonly using, link to the Wikipedia page for a description of the distribution, and list some common use cases for each one.

Reply:
Another important piece is choosing a strategy for coupling the parameters you're modeling, i.e. specifying an appropriate joint distribution, not just the marginals.  This is related to the bit about common random numbers, but I think it's a more general problem (I haven't thought carefully about how to do this in a systematic way within the Vivarium framework).

Reply:
I added an outline for all of these topics in the latest commit. Thanks for the ideas!